packages/lesswrong/components/comments/RecentDiscussionThread.jsx

```diff
@@ -172,7 +172,7 @@ class RecentDiscussionThread extends PureComponent {
             {nestedComments.map(comment =>
               <div key={comment.item._id}>
                 <CommentsNode
-                  startThreadTruncated={true}
+                  startThreadTruncated={(currentUser && currentUser.noCollapseCommentsFrontpage)}
```

Comment:
This seems backwards? If this change is just to add a setting (without changing the logged-out behavior), it should be `!(currentUser && currentUser.noCollapseCommentsFrontpage)`.